content/docs/guides/components/istio.md

```diff
@@ -60,12 +60,12 @@ so this service should not be of type `LoadBalancer`.
 - Edit `install/kubernetes/istio-demo.yaml` and change the service type of `istio-ingressgateway` from
   `LoadBalancer` to `NodePort`.
 
-### 4. Install istio (without mTLS)
+### 5. Install istio (without mTLS)
     ```
     kubectl apply -f install/kubernetes/istio-demo.yaml
     ```
 
-### (optional) 5. Deploy the Gateway
+### 6. Deploy the Gateway
 
 This is for rolling out model and doing traffic split. See more detail below.
 
```

Comment:
There's still a "TODO" here - can we add instructions for deploying the gateway now, since we've added line 109 which says "Requirement: Gateway is deployed, see above".

Reply:
updated, thanks